app/assets/javascripts/provider/admin/apiconfig/services/service_discovery.js

```diff
@@ -57,10 +57,15 @@ document.addEventListener("DOMContentLoaded", function(e) {
     opt.text = val;
     return selectElem.appendChild(opt);
   };
+  
+  function safe_fetch(url, init) {
+    const tempRequest = new Request(url, init);
+    return fetch(tempRequest, { credentials: "same-origin" });
+  }
 
   function fetchData(url, type) {
     if ('fetch' in window) {
-      fetch(url)
```

Comment:
or just pass the options here

Reply:
Agree, perhaps the simplest is better here.

Reply:
I agree, we can pass the credentials here, is a lot simpler:

`fetch(url, {
  credentials: 'same-origin'  
})`

Reply:
Ok, since there is only one point where it is used.

Reply:
Thanks @juazugas 
In fact, in other parts of the app we are already using the fetch polyfill.
We are incorporating the polyfill step by step, and this file is one of the next ones to be updated.